docker-scripts/build_all.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# exit when any command fails
+set -e
+
+START_DIR="$(pwd)"
+ROOT_DIR="$(dirname "$(dirname "$(readlink -fm "$0")")")"
+
+# Build base development images
+cd $ROOT_DIR/development/docker/
+./build.sh
+
+# Build jenkins-node using previous images
+cd $ROOT_DIR/jenkins-node/docker-images/os-builder/
+./build.sh
+
+# Build static analysis images
+cd $ROOT_DIR/jenkins-node/docker-images/static-analysis/
+./build.sh
+
+cd $START_DIR
```

Comment:
I'm not sure we need to switch back as the bash subprocess will exit leaving the parent cwd unchanged?

Reply:
Didn't know that, there was one script env (probably cmd) which dropped you in the last dir the script ran in, so I got into the habit of putting it in

Reply:
Yeah you're right Windows cmd/bat scripts do that and apparently if you do `source script.sh` that will affect the current shell process too.